content/docs/pipelines/reference/api/kubeflow-pipeline-api-spec.html

```diff
@@ -263,32 +276,38 @@ <h3 id="tag-JobService" class="swagger-summary-tag">Tag: JobService</h3>
                 <tbody>
                     <tr>
                         <td><a href="#operation--apis-v1beta1-jobs-get">GET /apis/v1beta1/jobs</a></td>
-                        <td></td>
+                        <td><p>Find all jobs.</p>
+        </td>
                     </tr>
                     <tr>
                         <td><a href="#operation--apis-v1beta1-jobs-post">POST /apis/v1beta1/jobs</a></td>
-                        <td></td>
+                        <td><p>Create a new job.</p>
+        </td>
                     </tr>
                     <tr>
                         <td><a href="#operation--apis-v1beta1-jobs--id--get">GET /apis/v1beta1/jobs/{id}</a></td>
-                        <td></td>
+                        <td><p>Find a specific job by ID.</p>
+        </td>
                     </tr>
                     <tr>
                         <td><a href="#operation--apis-v1beta1-jobs--id--delete">DELETE /apis/v1beta1/jobs/{id}</a></td>
-                        <td></td>
+                        <td><p>Delete a job.</p>
+        </td>
                     </tr>
                     <tr>
                         <td><a href="#operation--apis-v1beta1-jobs--id--disable-post">POST /apis/v1beta1/jobs/{id}/disable</a></td>
-                        <td></td>
+                        <td><p>Disable a job.</p>
```

Comment:
Is a kubeflow job analogous to a kubernetes job? I don't see anything describing them in the kubeflow documentation.

If they are the same, perhaps something like "Activate a job that performs a given series of runs."

Reply:
@OfficePop I updated the backend api version in swagger json. https://github.com/kubeflow/pipelines/pull/2770 If you re-generate the api spec html, it should be using 0.1.38 instead of 0.1.20 now.

Reply:
Updated as follows:

**Disable** - Stops a job and all its associated runs. The job is not deleted.
**Enable** - Restarts a job that was previously stopped. All runs associated with the job will continue.